### ISSUE TITLE
Create more examples

### ISSUE THREAD

**Issue by magicalcookie** [xforce/anno1800-mod-loader#4]
Would be nice to have some more examples 😸

**Comment by Smartertime** [comment]
Replace loots for world fair pool 9 : assets.xml in Anno 1800\mods\removeuselessloots\data\config\export\main\asset

`<ModOps>
	
    <ModOp Type="remove" Path="//Asset[Values/Standard/GUID='191256']/Values/RewardPool/ItemsPool/Item" />
    <ModOp Type="add" Path="//Asset[Values/Standard/GUID='191256']/Values/RewardPool/ItemsPool">
        <Item><ItemLink>190826</ItemLink></Item>
	</ModOp>
	
    <ModOp Type="remove" Path="//Asset[Values/Standard/GUID='191258']/Values/RewardPool/ItemsPool/Item" />
    <ModOp Type="add" Path="//Asset[Values/Standard/GUID='191258']/Values/RewardPool/ItemsPool">
        <Item><ItemLink>191896</ItemLink></Item>
	</ModOp>
	
    <ModOp Type="remove" Path="//Asset[Values/Standard/GUID='191257']/Values/RewardPool/ItemsPool/Item" />
    <ModOp Type="add" Path="//Asset[Values/Standard/GUID='191257']/Values/RewardPool/ItemsPool">
        <Item><ItemLink>190861</ItemLink></Item>
	</ModOp>
</ModOps>`

**Comment by kskudlik** [comment]
why don´t you just replace them?

```
<!-- Slot 1 -->
<ModOp Type="replace" Path="//Asset[Values/Standard/GUID='191256']/Values/RewardPool/ItemsPool/Item">
    <Item><ItemLink>190826</ItemLink></Item>
</ModOp>
<!-- Slot 2 -->
<ModOp Type="replace" Path="//Asset[Values/Standard/GUID='191258']/Values/RewardPool/ItemsPool/Item">
    <Item><ItemLink>191896</ItemLink></Item>
</ModOp>
<!-- Slot 3 -->
<ModOp Type="replace" Path="//Asset[Values/Standard/GUID='191257']/Values/RewardPool/ItemsPool/Item">
    <Item><ItemLink>190861</ItemLink></Item>
</ModOp>
```
that should do the trick. will try later :)

**Comment by Smartertime** [comment]
You're right, just tested it, works and is simpler. I started with a removal as it would be cleaner to compose more operations later.
